OSDOCS-17170 zero-trust pre-migration work (#104736)
openshift/openshift-docs: security/zero_trust_workload_identity_manager/zero-trust-manager-monitoring.adoc

:_mod-docs-content-type: ASSEMBLY
[id="zero-trust-manager-monitoring_{context}"]
= Monitoring {zero-trust-full}
include::_attributes/common-attributes.adoc[]
:context: zero-trust-manager-monitoring

toc::[]

[role="_abstract"]

> Configure monitoring to collect metrics from the Security Production Identity Framework for Everyone (SPIRE) Server and SPIRE Agent components. Collecting these metrics using the Prometheus Operator format helps you track the performance of the {zero-trust-full}.

Reviewer: ```suggestion
Track the performance of the {zero-trust-full} by collecting metrics. Configure monitoring to collect metrics from the Security Production Identity Framework for Everyone (SPIRE) Server and SPIRE Agent components. 
```

Author: Done
